src/i18n/en.json

```diff
@@ -690,23 +690,38 @@
     "actions.createDecision": "Create agreement",
     "actions.funds": "Funds",
     "actions.transferFunds": "Transfer funds",
+    "actions.description.transferFunds": "Transfer funds between teams ",
     "actions.mintTokens": "Mint tokens",
+    "actions.description.mintTokens": "Mint new tokens for the Colony",
     "actions.unlockToken": "Unlock token",
+    "actions.description.unlockToken": "Unlock new and approved tokens",
     "actions.manageTokens": "Manage tokens",
+    "actions.description.manageTokens": "Manage the tokens used in the Colony",
     "actions.teams": "Teams",
     "actions.createNewTeam": "Create new team",
+    "actions.description.createNewTeam": "Multi-phase projects with upfront allocation",
     "actions.editExistingTeam": "Edit a team",
+    "actions.description.editExistingTeam": "Multi-phase projects with upfront allocation",
     "actions.admin": "Admin",
-    "actions.manageVerifiedMembers": "Manage verified members",
+    "actions.manageReputation": "Manage reputation",
+    "actions.description.manageReputation": "Multi-phase projects with upfront allocation",
     "actions.managePermissions": "Manage permissions",
-    "actions.editColonyDetails": "Edit Colony details",
-    "actions.upgradeColonyVersion": "Upgrade Colony version",
+    "actions.description.managePermissions": "Multi-phase projects with upfront allocation",
+    "actions.editColonyDetails": "Edit colony details",
+    "actions.description.editColonyDetails": "Multi-phase projects with upfront allocation",
+    "actions.upgradeColonyVersion": "Upgrade colony version",
+    "actions.description.upgradeColonyVersion": "Multi-phase projects with upfront allocation",
+    "actions.manageVerifiedMembers": "Manage verified members",
+    "actions.description.manageVerifiedMembers": "Multi-phase projects with upfront allocation",
+    "actions.recoveryMode": "Enter recovery mode",
+    "actions.description.recoveryMode": "Multi-phase projects with upfront allocation",
```

Comment:
These match the figma designs, but not sure if @arrenv can provide the actual copy for these?